okonomiyaki/runtimes/runtime.py

```diff
@@ -98,7 +98,7 @@ def from_prefix_and_platform(cls, prefix, platform, version=None,
             scriptsdir = bindir = posixpath.join(prefix, "bin")
             paths = (bindir, )
 
-        if version.major == 3:
+        if version.major == 3 and platform.os_kind != OSKind.windows:
```

Comment:
@itziakos I think you can go ahead. 

@sjagoe I would guess from your comment that the `python3` link is guaranteed to be there!